quantum/keymap_extras/keymap_slovenian.h

```diff
@@ -89,6 +92,8 @@
 #define SI_UNDS LSFT(SI_MINS) // _
 
 // Alt Gr-ed characters
+
+=======
```

Comment:
This needs to be removed.

Reply:
Apologies, this is due to some conflicts in update from upstream